src/client/js/otp/config.js

```diff
@@ -316,6 +316,14 @@ otp.config.modes = {
     "CAR,WALK,TRANSIT"          : _tr('Kiss and Ride'),
     //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
     "BICYCLE_PARK,WALK,TRANSIT" : _tr('Bike and Ride'),
+    //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
+    "FLEX_ACCESS,WALK,TRANSIT" : _tr('Transit with flex access'),
+    //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
+    "FLEX_EGRESS,WALK,TRANSIT" : _tr('Transit with flex egress'),
+    //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
+    "FLEX_ACCESS,FLEX_EGRESS,TRANSIT" : _tr('Transit with flex access and egress'),
+    //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
+    "FLEX_DIRECT" : _tr('Direct flex search'),
```

Comment:
This should be listed after all bicycle options. This will also make the text appear in a more natural order in the translation files above.